### ISSUE TITLE
v7.3.24 not loading modules

### ISSUE THREAD

**Issue by Myrroddin** [BigWigsMods/LittleWigs#294]
```
2x Usage: GetModule(name, silent): 'name' - Cannot find module 'The Sand Queen'.
[C]: ?
Ace3\AceAddon-3.0\AceAddon-3.0-12.lua:243: in function <Ace3\AceAddon-3.0\AceAddon-3.0.lua:241>
(tail call): ?
BigWigs_Core\Core.lua:593: in function `AddColors'
...ceLittleWigs\BfA\TolDagor\Options\Colors-Colors.lua:2: in main chunk
[C]: ?
[C]: in function `LoadAddOn'
BigWigs\Loader.lua:300: in function <BigWigs\Loader.lua:288>
BigWigs\Loader.lua:310: in function <BigWigs\Loader.lua:307>
BigWigs\Loader.lua:320: in function <BigWigs\Loader.lua:318>
BigWigs\Loader.lua:1163: in function `?'
BigWigs\Loader.lua:893: in function <BigWigs\Loader.lua:892>

Locals:
(*temporary) = "Usage: GetModule(name, silent): 'name' - Cannot find module 'The Sand Queen'."
```

**Comment by nebularg** [comment]
Fixed with BigWigsMods/BigWigs@23cafd9

(leaving this open until there is a new BigWigs tags)

**Comment by Myrroddin** [comment]
Thank you. Even though the error clearly says BigWigs, not LittleWigs, I only get the error in 5 person dungeons, and haven't seen it in raids, hence me posting it here.

Again, thank you!
